### PR TITLE
camera trigger : multipin support

### PR DESCRIPTION
**Pull request by mhkabir**
Allows triggering on other AUX pins.

### CODE REVIEW

File: src/modules/camera_trigger/camera_trigger.cpp [hunk 1]
```diff
@@ -161,7 +169,7 @@ CameraTrigger::CameraTrigger() :
 {
 	memset(&_trigger, 0, sizeof(_trigger));
 	memset(&_sensor, 0, sizeof(_sensor));
-	memset(&_command, 0, sizeof(_command));
+	memset(&_command, 0, sizeof(_command));
```
Comment (LorenzMeier) [inline src/modules/camera_trigger/camera_trigger.cpp]: please run the fix code style script: ```Tool/fix_code_style.sh src/modules/camera_trigger/*.cpp```

**Comment by LorenzMeier** [comment]
Looks good, except for minor code style issue.

**Comment by mhkabir** [comment]
@LorenzMeier Done.
Following up in next weeks with a closed-loop active strobe/exposure control system.

**Comment by LorenzMeier** [comment]
Ok, looking great now! I presume this has been tested and validated in this state?

**Comment by LorenzMeier** [comment]
@mhkabir I'm still missing the feedback on testing state, and there seem now more commits in this branch which are unrelated. Could you comment on wether you have tested this in the system?

**Comment by mhkabir** [comment]
@LorenzMeier Some mess up with the branches, re-opening another PR with these changes. 
The changes are tested to work fine, but something seems to be crashing this again (after it was merged to master), trying to track it down. Maybe the FMU app is interfering.

**Pull request closed without merging**